cmd/xl-v1.go

```diff
@@ -245,3 +245,9 @@ func (xl xlObjects) crawlAndGetDataUsage(ctx context.Context, endCh <-chan struc
 
 	return dataUsageInfo
 }
+
+// GetMetrics - no op
+func (xl xlObjects) IsReady(ctx context.Context) bool {
+	logger.LogIf(ctx, NotImplemented{})
```

Comment:
this should be logger.CriticalIf() if trying to panic the code in case of incorrect usage

Reply:
Changed @harshavardhana :+1: